<launch>
    <arg name="port" default="/dev/ttyUSB0" />
    <arg name="baud" default="57600" />

    <node name="teleop_input" pkg="maniros" type="teleop_input.py" output="screen"/>
    <node name="cmd_vel_adapter" pkg="maniros" type="cmd_vel_adapter.py" output="screen"/>
    <node name="control_adapter" pkg="maniros" type="control_adapter.py" output="screen">
	<param name="robot_height" type="int" value="15" />
	<param name="robot_width" type="int" value="30" />
    </node>
    <node name="joy_node" pkg="joy" type="joy_node" output="screen">
	<param name="dev" type="string" value="/dev/input/js0" />
    </node>
    <node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen">
	<param name="port" type="string" value="$(arg port)" />
	<param name="baud" type="int" value="$(arg baud)" />
    </node>
</launch>
